src/targets/chrome/docker.js

```diff
@@ -96,9 +125,15 @@ function createChromeDockerTarget({
     );
     const { code, stdout, stderr } = await execa(dockerPath, args);
     if (code === 0) {
-      await waitOnCDPAvailable(port);
-      dockerId = stdout;
-      debug(`Docker started with id ${dockerId}`);
+      try {
+        dockerId = stdout;
+        host = await getNetworkHost(dockerId);
+        await waitOnCDPAvailable(host, port);
+        debug(`Docker started with id ${dockerId}`);
+      } catch (err) {
+        await stop();
```

Comment:
Do we have to do `stop()` here or will it be handled down the line regardless?

Reply:
I was seeing issues in Jenkins of failed processes not being killed, but let me investigate this further and do a separate PR with a targeted bug and fix.

Reply:
Sounds good, thx!